UnityProject/Assets/Scripts/Player/PlayerEffectsManager.cs

```diff
@@ -26,16 +26,16 @@ private void Update()
 	{
 		if(PlayerManager.PlayerScript.PlayerSync.isFloatingClient == true && floatingEffect.willAnimate == false)
 		{
-			AnimateFloating();
+			animateFloating();
 			return;
 		}
 		if(PlayerManager.PlayerScript.PlayerSync.isFloatingClient == false && floatingEffect.willAnimate == true)
 		{
-			AnimateFloating();
+			animateFloating();
 		}
 	}
 
-	public void AnimateFloating()
+	public void animateFloating()
```

Comment:
You did the opposite of PascalCase. I was referring to the public field you're using inside the if statement, ``willAnimate```is using camelCase when it should be PascalCase.